### PR TITLE
NAS-130605 / 25.04 / Various dashboard improvements

### PR DESCRIPTION
**Pull request by AlexKarpov98**
**Changes:**
Widget names changes, Widget type -> naming mapping, WIdget title alignment.

**Testing:**

See ticket, test each step.

### Downstream

|Affects         |Reasoning
|----------------|-------------------------------
|Documentation   | Widget names changes

### CODE REVIEW

File: src/app/pages/dashboard/components/dashboard/dashboard.component.scss [hunk 1]
```diff
@@ -89,9 +89,10 @@
     pointer-events: none;
   }
 
-  mat-card-content .header h3 {
+  mat-card-content .header h3,
+  .slot .wrapper h3 {
     @media(max-width: 767px) {
-      font-size: 20px;
+      font-size: 20px !important;
```
Comment (undsoft) [inline src/app/pages/dashboard/components/dashboard/dashboard.component.scss:95]: So, why do we need to do this via ng-deep, why can't we have this style the component that needs it? It's really hard to see what this style applies to.
Also can't you avoid `!important` with more specificity?

File: src/app/pages/dashboard/types/widget-type-labels.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { hostnamePassiveWidget } from 'app/pages/dashboard/widgets/system/widget-hostname-passive/widget-hostname-passive.definition';
+import { osVersionWidget } from 'app/pages/dashboard/widgets/system/widget-os-version/widget-os-version.definition';
+import { serialActiveWidget } from 'app/pages/dashboard/widgets/system/widget-serial-active/widget-serial-active.definition';
+import { serialPassiveWidget } from 'app/pages/dashboard/widgets/system/widget-serial-passive/widget-serial-passive.definition';
+import { systemInfoActiveWidget } from 'app/pages/dashboard/widgets/system/widget-sys-info-active/widget-sys-info-active.definition';
+import { systemInfoPassiveWidget } from 'app/pages/dashboard/widgets/system/widget-sys-info-passive/widget-sys-info-passive.definition';
+import { systemImageWidget } from 'app/pages/dashboard/widgets/system/widget-system-image/widget-system-image.definition';
+import { systemUptimeWidget } from 'app/pages/dashboard/widgets/system/widget-system-uptime/widget-system-uptime.definition';
+
+export const widgetTypeLabels = new Map<WidgetType, string>([
```
Comment (undsoft) [inline src/app/pages/dashboard/types/widget-type-labels.ts:30]: This creates another place that needs to be updated every time we add a widget. It's easy to forget to do so, as there won't be any errors.
I'd rather we try to keep widgets more self-contained and relied on widget definition and `widgetRegistry` for the name.

**Review by undsoft — CHANGES_REQUESTED** [review]

File: src/app/pages/dashboard/types/widget-type-labels.ts (hunk 2 above)
Comment (AlexKarpov98) [reply src/app/pages/dashboard/types/widget-type-labels.ts]: @undsoft - yeah, sorry I made it more complicated.

**Review by undsoft — APPROVED** [review]
Looks good, thanks.

**Pull request merged**